index.js

```diff
@@ -20,11 +20,11 @@ module.exports = {
 		return nodeNightlyVer.then(latest => {
 			const conf = new Configstore(pkg.name);
 			conf.set('version', latest);
-			const os = process.platform;
+			const os = process.platform  === 'win32' ? 'win' : process.platform;
 			const arch = process.arch;
 			const type = 'nightly';
 			osArchString = `${latest}-${os}-${arch}`;
-			const url = `https://nodejs.org/download/${type}/${latest}/node-${osArchString}.tar.gz`;
+			const url = `https://nodejs.org/download/${type}/${latest}/node-${osArchString}.zip`;
```

Comment:
Oh Got it! :+1: 
You mean to say like '.zip' for 'win' and '.tar.gz' for other? [ can be done using if/else]